MapboxDirections/MBRouteOptions.swift

```diff
@@ -195,6 +202,7 @@ open class RouteOptions: NSObject, NSSecureCoding {
         coder.encode(routeShapeResolution.description, forKey: "routeShapeResolution")
         coder.encode(attributeOptions.description, forKey: "attributeOptions")
         coder.encode(includesExitRoundaboutManeuver, forKey: "includesExitRoundaboutManeuver")
+        coder.encode(excludeRoadClasses, forKey: "excludeRoadClasses")
```

Comment:
Encode `excludedRoadClasses.description`, which is a string, not `excludeRoadClasses`, whose underlying type is `Int` and thus more fragile.